src/ws-tiles-chart/ws-tiles-chart.js

```diff
@@ -24,7 +29,10 @@ export class WSTilesChart extends Component {
     maxTileSize: 25,
     minTileSize: 8,
     width: 80,
-    height: 80
+    height: 80,
```

Comment:
I was not involved in the Design talks. This is a component that the UX designed for one of our applications and I implemented while creating it. I added the default size on my own, I did not ask it from him. I think we will need it in different sizes for our application. But maybe we could ask a default from them.